booking/src/main/java/com/projectlab/travelin_android/booking/MyClass.kt

```diff
@@ -0,0 +1,4 @@
+package com.projectlab.travelin_android.booking
+
+class MyClass {
+}
```

Comment:
I keep seeing this class on several PRs, let's make sure not to add empty or unused classes